README.md

```diff
@@ -12,12 +12,14 @@ This repository contains the code for ACM Hack’s website!
 This website is developed with [Gatsby](https://www.gatsbyjs.org/) and
 [Material UI](https://material-ui.com/), and is an active project being
 developed by ACM Hack’s Dev Team:
-Alex Xia [(@khxia)](https://github.com/khxia/),
-Jody Lin [(@jodymlin)](https://github.com/jodymlin/),
 Nareh Agazaryan [(@nareha)](https://github.com/nareha),
+Katelyn Yu [(@katelynsyu)](https://github.com/katelynsyu),
 Einar Balan [(@EinarBalan)](https://github.com/EinarBalan),
 Timothy Gu [(@TimothyGu)](https://github.com/TimothyGu/),
```

Comment:
:wave: Please feel free move me (and other folks) to a separate "alumni" section or something like that :) I'm clearly not part of the ACM Hack Dev Team anymore.

Reply:
You'll always be a part of dev team in our hearts 🥹